submodules/CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ endfunction()
 # Add taraxa-vdf
 set(VDF_LIBRARY_COMMAND "\
  ${CMAKE_MAKE_PROGRAM}\
- CPPFLAGS=\"-I./include -I${CONAN_INCLUDE_DIRS_OPENSSL} -I${CONAN_INCLUDE_DIRS_GMP} -I${CONAN_INCLUDE_DIRS_MPFR} -std=c++17\"\
+ CPPFLAGS=\"-I./include -I${CONAN_INCLUDE_DIRS_OPENSSL} -I${CONAN_INCLUDE_DIRS_GMP} -I${CONAN_INCLUDE_DIRS_MPFR} -std=c++17 -O3\"\
```

Comment:
isn't this set automatically according to CMAKE_CXX_FLAGS_RELEASE var ?

Reply:
not in our case, as we are using special script `setenv_for_taraxa_vdf.sh ` that is generated by this command. So we completely overwrite whatever is in CPPFLAGS

Reply:
aah ok